libs-scala/timer-utils/src/main/scala/com/digitalasset/timer/RetryStrategy.scala

```diff
@@ -30,12 +30,12 @@ object RetryStrategy {
     new RetryStrategy(Some(attempts), waitTime, waitTime, identity, { case _ => true })
 
   /**
-    * Retry indefinitely with constant wait time, but only if the exception satisfies a predicate
-    *
+    * Retry with constant wait time, but only if the exception satisfies a predicate
     */
-  def constant(waitTime: Duration)(predicate: PartialFunction[Throwable, Boolean]): RetryStrategy =
-    new RetryStrategy(None, waitTime, waitTime, identity, predicate)
-
+  def constant(attempts: Option[Int] = None, waitTime: Duration)(
```

Comment:
I would say that making a overloading would make for a more safe and readable alternative than using an `Option` with a default.

Reply:
Tried that but it collides with the other overload of `constant`. I'd have to pick a new name, and I didn't want to because I am lazy and couldn't think of one. (If you have one, please shout.)

Reply:
We're using it only in one place, I guess it's fine like it is then. Thanks.